code/src/components/AddTask.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import styled from 'styled-components'
+
+import { tasks } from '../reducers/tasks'
+
+const AddTaskContainer = styled.form`
```

Comment:
Hey Sara, it is cool that you decided to use styled components and this one looks really neat!